### PR TITLE
Delete all selected lines using the "delete line" shortcut in script editor

### PR DESCRIPTION
**Pull request by Yanpas**
Currently hotkey deletes only single line in an editor, now all selected lines are deleted

### CODE REVIEW

**Comment by groud** [comment]
Thanks for your contribution.
Would you mind [amend your commit](http://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) with a more descriptive message ?

Also it seems that the email address you used to author you commit is not the same as your Github address, thus it look in Github that you're not the commit author. You might want to change the commit's author.

**Comment by Yanpas** [comment]
thanks for the tips, done :)

**Comment by Noshyaar** [comment]
Thank you. There are some styling issues. The changes didn't comply with clang-format. Please use `clang-format -i <filename>` and `git commit --amend`. ([PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request))

**Comment by Yanpas** [comment]
done

File: editor/plugins/script_text_editor.cpp [hunk 1]
```diff
@@ -935,13 +935,27 @@ void ScriptTextEditor::_edit_option(int p_op) {
...
 			tx->begin_complex_operation();
-			int line = tx->cursor_get_line();
-			tx->set_line(tx->cursor_get_line(), "");
-			tx->backspace_at_cursor();
-			tx->unfold_line(line);
-			tx->cursor_set_line(line);
+			if (tx->is_selection_active()) {
+				int to_line = tx->get_selection_to_line();
+				int from_line = tx->get_selection_from_line();
+				int count = abs(to_line - from_line) + 1;
```
Comment (Noshyaar) [inline editor/plugins/script_text_editor.cpp]: This should be `Math::abs(...)` (or `ABS` macro.)

**Pull request merged**